docs/howtoguides/use_custom_apt_config.rst

```diff
@@ -0,0 +1,99 @@
+How to use the Pro Client with custom APT configuration
+*******************************************************
+
+The Pro Client CLI is able, through the security-status command, to show
+information about installed packages in the system, their origins, and
+potential updates that may be available for these. This information is also
+available through the
+:ref:`packages updates API<references/api:u.pro.packages.updates.v1>` and the
+:ref:`packages summary API<references/api:u.pro.packages.summary.v1>` calls,
+for example.
+
+The information presented on the outputs mentioned above is taken directly
+from the local APT caches. Packages, origins, versions, candidates, all is
+calculated using the ``python3-apt`` library, which in turn relies on whatever
+APT has knowledge about in a particular machine.
+
+Some users may have the need to configure APT to some specific behaviour. An
+example of a use case would be to pass custom ``.list`` or ``.sources`` files,
+to APT, whether to limit or to increase knowledge about different package
+sources.
+
+The Pro Client will respect the `configuration set directly on APT`_, at
+execution time. If system level configuration is present, it will be used by
+default. To configure APT without changing the system, there is the option of
+passing the config file as an environment variable.
+
+Use an APT config file
+=========================
+For example, consider that the ``/etc/apt/apt.conf.d/50-custom`` file exists,
```

Comment:
```suggestion
=========================

For example, consider that the ``/etc/apt/apt.conf.d/50-custom`` file exists,
```

Reply:
rST likes to have a blank line after headers :)

Reply:
wah rST can mind its own business I write no useless lines

Reply:
joking, will fix it.